### ISSUE TITLE
Missing Dockerfile

### ISSUE THREAD

**Issue by ericfrederich** [theNetworkChuck/NetworkChuck#10]
Any chance of publishing the Dockerfile for `thenetworkchuck/networkchuck_pihole`?

Someone else was asking to update the version of PiHole as well.
If this was published, I'm sure you'd even get merge requests for it.

**Comment by dubbleb88** [comment]
I found these settings in the python file, is it what you are looking for?

docker run -d \
    --name pihole \
    -p 53:53/tcp -p 53:53/udp \
    -p 80:80 \
    -p 443:443 \
    -p 8080:8080 \
    -e TZ="America/Chicago" \
    -v "$(pwd)/etc-pihole/:/etc/pihole/" \
    -v "$(pwd)/etc-dnsmasq.d/:/etc/dnsmasq.d/" \
    --dns=127.0.0.1 --dns=1.1.1.1 \
    --restart=unless-stopped \
    thenetworkchuck/networkchuck_pihole

Also, I was having a dms resolution issue and used this page to resolve and update my pihole version:

https://discourse.pi-hole.net/t/dns-service-not-running/13785

**Comment by ericfrederich** [comment]
> I found these settings in the bash file, is it what you are looking for?

No, that's how to invoke the `thenetworkchuck/networkchuck_pihole` image, but not the source code for it.
If the `Dockerfile` were published someone could do a `docker build` and get an up to date PiHole version without waiting for @theNetworkChuck to do it.

**Comment by ericfrederich** [comment]
Made one myself, improves on several things.  Next I need to publish it to Dockerhub

https://github.com/ericfrederich/fancy-pihole

**Comment by wiredyeti** [comment]
I dont know how to use that @ericfrederich
